src/EPR.Calculator.API/Controllers/CalculatorController.cs

```diff
@@ -52,5 +55,33 @@ public IActionResult GetCalculatorRun()
             // TODO: Return the details of a particular run
             return new OkResult();
         }
+
+        [HttpGet]
+        [Route("calculatorRunByName/{name}")]
+        public IActionResult GetCalculatorRunByName([FromRoute] string name)
+        {
+            if (!ModelState.IsValid)
+            {
+                return StatusCode(StatusCodes.Status400BadRequest, ModelState.Values.SelectMany(x => x.Errors));
+            }
+
+            var calculatorRun = _context.CalculatorRuns.Where(run => run.Name.ToLower() == name.ToLower());
```

Comment:
Should this be 
`_context.CalculatorRuns.SingleOrDefault(run => run.Name.ToLower() == name.ToLower());`
If no result it should return 404